zk_toolbox/crates/zk_inception/src/consts.rs

```diff
@@ -12,27 +10,6 @@ pub const DEFAULT_UNSIGNED_TRANSACTIONS_DIR: &str = "transactions";
 pub const BELLMAN_CUDA_DIR: &str = "era-bellman-cuda";
 pub const L2_BASE_TOKEN_ADDRESS: &str = "0x000000000000000000000000000000000000800A";
 
-#[allow(non_upper_case_globals)]
-const kB: usize = 1024;
-
-/// Max payload size for consensus in bytes
-pub const MAX_PAYLOAD_SIZE: usize = 2_500_000;
-/// Max batch size for consensus in bytes
-/// Compute a default batch size, so operators are not caught out by the missing setting
-/// while we're still working on batch syncing. The batch interval is ~1 minute,
-/// so there will be ~60 blocks, and an Ethereum Merkle proof is ~1kB, but under high
-/// traffic there can be thousands of huge transactions that quickly fill up blocks
-/// and there could be more blocks in a batch then expected. We chose a generous
-/// limit so as not to prevent any legitimate batch from being transmitted.
-pub const MAX_BATCH_SIZE: usize = MAX_PAYLOAD_SIZE * 5000 + kB;
```

Comment:
We can remove because of:
https://github.com/matter-labs/zksync-era/blob/eed8198f97fd0606884271c86eebd700785ccbd8/core/lib/protobuf_config/src/consensus.rs#L139